release/remote.py

```diff
@@ -44,31 +32,43 @@ def download_deployment_tarball(config: Config, system: System) -> ActionReturn:
 
 
 def publish_bokehjs_to_cdn(config: Config, system: System) -> ActionReturn:
-    subdir = "dev" if V(config.version).is_prerelease else "release"
     version = config.version
+    subdir = "dev" if V(version).is_prerelease else "release"
 
     access_key_id = config.secrets["AWS_ACCESS_KEY_ID"]
     secret_access_key = config.secrets["AWS_SECRET_ACCESS_KEY"]
 
+    file_names = ("bokeh", "bokeh-gl", "bokeh-api", "bokeh-widgets", "bokeh-tables", "bokeh-mathjax")
+    suffixes = ("js", "min.js", "esm.js", "esm.min.js")
+
     try:
-        buckets = []
-        for bucket_name, bucket_region in [("cdn.bokeh.org", "us-east-1"), ("cdn-backup.bokeh.org", "us-west-2")]:
-            conn = boto.s3.connect_to_region(
-                bucket_region,
-                aws_access_key_id=access_key_id,
-                aws_secret_access_key=secret_access_key,
-                calling_format=boto.s3.connection.OrdinaryCallingFormat(),
-            )
-            buckets.append(conn.get_bucket(bucket_name))
-
-        content_type = "application/javascript"
-        for name in ("bokeh", "bokeh-gl", "bokeh-api", "bokeh-widgets", "bokeh-tables", "bokeh-mathjax"):
-            for suffix in ("js", "min.js", "esm.js", "esm.min.js"):
-                local_path = f"bokehjs/build/js/{name}.{suffix}"
-                cdn_path = f"bokeh/{subdir}/{name}-{version}.{suffix}"
-                for bucket in buckets:
-                    _upload_file_to_cdn(local_path, cdn_path, content_type, bucket)
+        for name, suffix in product(file_names, suffixes):
+            local_path = f"bokehjs/build/js/{name}.{suffix}"
+            cdn_path = f"bokeh/{subdir}/{name}-{version}.{suffix}"
+
+            with open(local_path) as f:
+                data = f.read().encode("utf-8")
+
+            for bucket, region_name in BOKEHJS_BUCKETS:
+                s3 = boto3.client(
+                    "s3",
+                    region_name=region_name,
+                    aws_access_key_id=access_key_id,
+                    aws_secret_access_key=secret_access_key,
+                )
+
+                LOG.record(f":uploading to CDN: {cdn_path}")
```

Comment:
```suggestion
                LOG.record(f":uploading to CDN {bucket}: {cdn_path}")
```

Otherwise, there will be two identical logs for each bucket.

Reply:
https://github.com/bokeh/bokeh/pull/14517/commits/96bdbcde049580da33146e282274264b926eed91